honeygrove/core/HoneyAdapter.py

```diff
@@ -28,6 +28,9 @@ def init():
 
         BrokerEndpoint.startListening()
 
+        if config.init_peer:
+            BrokerEndpoint.peerTo(config.init_peer_ip, config.init_peer_port)
```

Comment:
Is this save if the `config.init_peer_ip` is not responsive?
Is the `init()` stuck on the peering, throws exception or continues unpeered after a timeout?

Reply:
There only will be error messages in the logs, every time Broker tries to connect to the peer.
You can safely remove and correct peering with the management console afterwards.